src/gateway/events/Connection.ts

```diff
@@ -30,6 +30,10 @@ export async function Connection(
 
 	socket.ipAddress = ipAddress;
 
+
+	const session_id = genSessionId();
+	socket.session_id = session_id; //Set the session of the WebSocket object
```

Comment:
Please read our [contributing guide](https://docs.fosscord.com/contributing/). Specifically
> Leave comments in your code about why something is done when appropriate, not just what it is doing.

Thank you for your contribution, however.

/j

Reply:
Comment added, not sure why its not showing up as outdated.